index.html

```diff
@@ -13,44 +13,50 @@
     <title>Thrive Launcher</title>
   </head>
   <body>
-    
+
+    <span class="BottomFade"
+      style="height: 5%; width: calc(35% + 5px); bottom: calc(70px + 22px); left: 8px;">
+    </span>
+
     <span class="ContentBlock"
-          style="height: 90%; width: 40%; top: 0px; left: 0px; border-right: 1px solid 
+          style="height: calc(80% + 10px); width: calc(35% + 5px); top: 10px; left: 8px;
                  rgba(68, 68, 68, 0.46)">
 
-      <h2>Development Forums</h2>
+      <h2>DEVELOPMENT FORUMS</h2>
       <hr>
 
       <span id="devForumPosts">
       </span>
     </span>
 
+    <span class="BottomFade"
+      style="height: 5%; width: calc(60% + 12px); bottom: calc(70px + 22px); left: calc(35% + 25px);">
+    </span>
+    
     <span class="ContentBlock"
-          style="height: 90%; width: calc(60% - 1px); top: 0px; left: calc(40% + 1px);">
+          style="height: calc(80% + 10px); width: calc(60% + 12px); top: 10px; left: calc(35% + 25px)">
 
-      <h2>News</h2>
+      <h2 style="padding-left: 50px;">NEWS</h2>
```

Comment:
Another capitalisation change that I find myself perplexed by, along with the indentation. In my opinion, this doesn't look very good when viewed in the launcher. Why?

Reply:
Also the same reason as before. The original uncapitalized one looks fine though, i could change to that.